subsys/bluetooth/host/gatt.c

```diff
@@ -1475,20 +1499,45 @@ static u8_t notify_cb(const struct bt_gatt_attr *attr, void *user_data)
 	return BT_GATT_ITER_CONTINUE;
 }
 
-int bt_gatt_notify_cb(struct bt_conn *conn, const struct bt_gatt_attr *attr,
-		      const void *data, u16_t len,
-		      bt_gatt_complete_func_t func, void *user_data)
+static u8_t match_uuid(const struct bt_gatt_attr *attr, void *user_data)
+{
+	struct bt_gatt_attr **found = user_data;
```

Comment:
Shouldn't this variable type have a `const` somewhere so you don't need the typecast below?

Reply:
Right, will fix it.

Reply:
Fixed.